### PR TITLE
CRT: Add subheading for clarity

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation task: https://mattermost.atlassian.net/browse/MM-37112

Updated:
- Messaging > Work with Messages > Organizing Conversations using Collapsed Reply Threads (Beta)
   - Added subheading **Enable Collapsed Reply Threads**

### CODE REVIEW

**Review by justinegeffen — APPROVED** [review]

File: source/messaging/organizing-conversations.rst [hunk 1]
```diff
@@ -7,8 +7,13 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
...
 .. image:: ../images/collapsed-reply-threads.gif
   :alt: Organize conversations using Collapsed Reply Threads.
+  
+Enable Collapsed Reply Threads
+------------------------------
 
-Your System Admin must enable the Collapsed Reply Threads feature in the System Console. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta>`__ documentation for details. You can `enable this feature for your Mattermost account <https://docs.mattermost.com/help/settings/account-settings.html#collapsed-reply-threads-beta>`__ by going to **Account Settings > Display > Collapsed Reply Threads (Beta)**.
+Your System Admin must enable the Collapsed Reply Threads feature in the System Console. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta>`__ documentation for details. 
+
+Once this feature is enabled in the System Console, you can `enable this feature for your Mattermost account <https://docs.mattermost.com/help/settings/account-settings.html#collapsed-reply-threads-beta>`__ by going to **Account Settings > Display > Collapsed Reply Threads (Beta)**.
```
Comment (amyblais) [inline source/messaging/organizing-conversations.rst:16]: @cwarnermm Doesn't seem to be related to this PR, but the links here don't get directed to the correct section in the docs. (e.g. clicking https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta just goes to https://docs.mattermost.com/configure/configuration-settings.html)

Comment (cwarnermm) [reply source/messaging/organizing-conversations.rst:16]: Thanks, @amyblais! I've fixed these broken links via https://github.com/mattermost/docs/pull/4786
